app/models/notification.rb

```diff
@@ -0,0 +1,31 @@
+class Notification < ActiveRecord::Base
+  acts_as_readable on: :updated_at
+  belongs_to :user
+  belongs_to :course
+
+  scope :center_popup, -> { where(type: 'Notification::CenterPopup') }
```

Comment:
This is not how to use STI. The direct class should be instantiated and used directly.